autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/CourseSelectCard.tsx

```diff
@@ -46,6 +49,10 @@ const CourseSelectCard: React.FC<CourseSelectCardProps> = ({ id }) => {
       role="button"
       tabIndex={0}
       onKeyPress={toggleCollapsed}
+      style={{
+        backgroundColor: getDisabledCourseCardColor(disabled),
+      }}
+      data-testid="card-header"
```

Comment:
Is there a particular reason other than semantics? I wouldn't want someone that uses screen readers to read "card-header" since it doesn't actually help them navigate the page I don't think

Reply:
No, that's just how we've been handling other test ids so I wanted to make it consistent. But unlike the others that were converted to aria labels since they actually made sense like that (they had functionality illustrated by the labels), now that you point it out this doesn't help with readability.

Honestly I'm fine with it being left as-is, if a test-id would make sense as an aria label it should be made one but in cases like this I don't mind it